### ISSUE TITLE
EPUB3 Media Overlays + TTS reada aloud, missing keyboard shortcuts (notably: right and left arrow keys, playback control)

### ISSUE THREAD

**Issue by ManfredMuchenberger** [edrlab/thorium-reader#1069]
IThorium v1.3.1-alpha.1.40
Windows 10

If during playback of a book with media overlays press the right arrow key, the focus (highlighted word) and also the spoken audio often jumps back to the beginning of a chapter.

Additional question: what exactly should be the "unit" or the rules of a jump if I press the right arrow key?

**Comment by danielweck** [comment]
Keyboard shortcuts for EPUB3 Media Overlays are currently not implemented (also not supported for TTS readaloud).

The left and right keyboard arrow keys are currently bound to the previous / next commands of "page" units (when documents are paginated using CSS Columns), or scroll units (during vertical scroll mode), or document units (fixed layout publications).

See this wiki page for the default keyboard shortcuts preset (which can be edited in the application settings, via the main library window):
https://github.com/edrlab/thorium-reader/wiki/Keyboard-shortcuts

Let's keep this issue open, I am just going to adjust the title.

**Comment by danielweck** [comment]
Related issue: https://github.com/edrlab/thorium-reader/issues/1107
